package.json

```diff
@@ -48,6 +48,7 @@
   "devDependencies": {
     "benchmark": "^2.1.4",
     "bluebird": "^3.5.4",
+    "chai-as-promised": "^7.1.1",
```

Comment:
You may remove this unused package ```chai-as-promised``` , I pulled your changes and all local tests are passing without it.